.expeditor/config.yml

```diff
@@ -0,0 +1,7 @@
+---
+project:
+  alias: core-plans
```

Comment:
I don't think we need this.

Reply:
My understanding was alias will allow other projects to refer to this as `core-plans` rather than `habitat-sh/core-plans#master` in their configuration which I _think_ we want.  Happy to change if my understanding of aliases is wrong.